addons/account/models/account_move_line.py

```diff
@@ -501,7 +501,7 @@ def _compute_account_id(self):
                        AND line.id != ANY(%(current_ids)s)
                 ),
                 properties AS(
-                    SELECT DISTINCT ON (property.company_id, property.name)
+                    SELECT DISTINCT ON (property.res_id, property.name)
```

Comment:
Probably wrong, because you can different property's for the same res_id for different companies.

Reply:
Would adding `SELECT DISTINCT ON (property.company_id, property.res_id, property.name)` be better ?